src/components/AvatarWithImagePicker.tsx

```diff
@@ -323,20 +322,22 @@ function AvatarWithImagePicker({
                             style={[styles.pRelative, avatarStyle]}
                             ref={anchorRef}
                         >
-                            <View>
-                                {source ? (
-                                    <Avatar
-                                        containerStyles={avatarStyle}
-                                        imageStyles={[avatarStyle, styles.alignSelfCenter]}
-                                        source={source}
-                                        fallbackIcon={fallbackIcon}
-                                        size={size}
-                                        type={type}
-                                    />
-                                ) : (
-                                    <DefaultAvatar />
-                                )}
-                            </View>
+                            <OfflineWithFeedback pendingAction={pendingAction}>
```

Comment:
Why do we have two OfflineWithFeedback?

Reply:
Having two `OfflineWithFeedback` for the same underlaying component feels like a workaround. Can you get ride of one of them and try to fix the error message? (try use styles.w100 if it helps)

Reply:
@nexarvo Let's not block on that. Can you just remove the extra wrapping View that's inside `<OfflineWithFeedback />`